spec/relativetimeformat.html

```diff
@@ -124,22 +124,15 @@ <h1>MakePartsList ( _pattern_, _unit_, _parts_ )</h1>
         </pre>
       </p>
       <emu-alg>
+        1. Let _patternParts_ be PartitionPattern(_pattern_).
         1. Let _result_ be a new empty List.
-        1. Let _beginIndex_ be ! Call(%StringProto_indexOf%, _pattern_, « "{", *0* »).
-        1. Let _length_ be the number of elements in _pattern_.
-        1. Repeat, while _beginIndex_ is an integer index into _pattern_
-          1. Set _endIndex_ to ! Call(%StringProto_indexOf%, _pattern_, « "}", _beginIndex_ »).
-          1. Assert: _endIndex_ is not -1, otherwise the pattern would be malformed.
-          1. If _beginIndex_ is greater than _nextIndex_, then
-            1. Let _literal_ be a substring of _pattern_ from position _nextIndex_, inclusive, to position _beginIndex_, exclusive.
-            1. Add new part Record { [[Type]]: `"literal"`, [[Value]]: _literal_ } as a new element of the list _result_.
-          1. Let _p_ be the substring of _pattern_ from position _beginIndex_, exclusive, to position _endIndex_, exclusive.
-          1. Assert: _p_ is `"0"`.
-          1. For each _part_ in _parts_, do
-            1. Add new part Record { [[Type]]: _part_.[[Type]], [[Value]]: _part_.[[Value]], [[Unit]]: _unit_ } as a new element on the List _result_.
-        1. If _nextIndex_ is less than _length_, then
-          1. Let _literal_ be the substring of _pattern_ from position _nextIndex_, exclusive, to position _length_, exclusive.
-          1. Add new part Record { [[Type]]: `"literal"`, [[Value]]: _literal_ } as a new element of the list _result_.
+        1. For each element _patternPart_ of _patternParts_, in List order, do
+          1. If _patternPart_.[[Type]] is `"literal"`, then
+            1. Append Record { [[Type]]: `"literal"`, [[Value]]: _patternPart_.[[Value]] } to _result_.
+          1. Else,
+            1. Assert: _patternPart_.[[Type]] is `"0"`.
```

Comment:
Am I correct that `"0"` here comes from the requirement for the pattern to have only one placeable names `0`? If that's true it may be worth documenting that requirement as well.

Reply:
Should I make a separate PR for it since it's editorial?

Reply:
I separated it out as a new issue.